docs/guides/anomaly-detection-configuration/timestamp-column.mdx

```diff
@@ -14,6 +14,11 @@ Elementary anomaly detection tests will use this column to create time buckets a
 It is highly recommended to configure a timestamp column (if there is one).
 The best column for this would be an `updated_at`/`created_at`/`loaded_at` timestamp for each row (date type also works).
 
+- When you specify a `timestamp_column`, when the test runs it splits the data to buckets according to the timestamp in this column, calculates the metric for each bucket and checks for anomalies between these buckets.
+  This also means that if the table has enough historical data, the test can start working right away.
+- When you do not specify a `timestamp_column`, each time the test runs it calculates the metric for all of the data in the table, and checks for anomalies between the metric from previous runs.
+  This also means that it will take the test `days_back` days to start working, as it needs to the time to collect the necessary metrics.
```

Comment:
It's actually `min_training_set_size` and not days back. 
For example - for hourly buckets the default `days_back` is 14 days but the default `min_training_set_size` is also 14, so the test can work after less than a day.